js/interface.js

```diff
@@ -555,6 +571,217 @@ function uploadFiles(files) {
   });
 }
 
+// FUNCTIONS FOR MOVING FILES/FOLDERS IN MODAL
+
+// Check if folder that has selected items to move has a subfolder and filter it
+function excludeFolders(items) {
+  var parentItemIdOfSelectedItems = navStack[navStack.length-1].id;
+  var selectedIdPlaceInModal = foldersForMoving.navStack[foldersForMoving.navStack.length-1].id;
+  var searchResult = items;
+  var selectedIdForMoving = [];
+  
+  for (var i = 0; i < selectedItemsForMoving.length; i++) {
+    if ($(selectedItemsForMoving[i]).attr('data-file-type') === 'folder') {
+      selectedIdForMoving.push(Number($(selectedItemsForMoving[i]).attr('data-id')));
+    }
+  }
+  
+  if (parentItemIdOfSelectedItems === selectedIdPlaceInModal) {
+    searchResult = searchResult.filter(function (item) {
+      for (var i = 0; i < selectedIdForMoving.length; i++) {
+        return item.id !== selectedIdForMoving[i];
+      }
+    })
+  }
+  
+  return searchResult;
+  
+}
+
+// Add app select item template
+function addAppsToSelect(apps) {
+  var $appSelectList = $('#organization-optgroup-' + apps.organizationId);
+  $appSelectList.append(templates.appsSelect(apps));
+}
+
+// Create list of folders in modal
+function addFoldersToMoveModal(res) {
+  if (Array.isArray(res)) {
+    if (res.length < 1) {
+      $('.move-modal-empty-state').addClass('active');
+    } else {
+      res = _.sortBy(res, [function(o) {
+        return o.name;
+      }]);
+      res.forEach(function (folder) {
+        $('.move-modal-list').append(templates.moveModalFolders(folder));
+      });
+    }
+  } else {
+    $('.move-modal-list').append(templates.moveModalFolders(res));
+    $('.move-modal-empty-state').removeClass('active');
+  }
+}
+
+// Add organization select item template
+function addOrganizationsToSelect(org) {
+  var $organizationsSelectList = $('#move-model-organizations-select');
+  $organizationsSelectList.append(templates.organizationsSelect(org));
+  appsList.forEach(addAppsToSelect);
+}
+
+// Users selected folders tree (for breadcrumbs)
+function addPathToStack(id, name, orgId, appId, parentId, type) {
+  foldersForMoving['navStack'].push(
+    {
+      id: Number(id)?Number(id):null,
```

Comment:
Have you read the code guidelines? It should be `a ? b : c` (note the spaces)

Note: you can also change this operation to `Number(id) || null`

Reply:
fixed